src/blister.c

```diff
@@ -284,17 +284,30 @@ Obj DoCopyBlist(Obj list, Int mut) {
   
 }
 
+Obj CopyBlistImm (
+    Obj                 list,
+    Int                 mut )
+{
+  return list;
+}
+		  
 Obj CopyBlist (
     Obj                 list,
     Int                 mut )
 {
-
-    /* don't change immutable objects                                      */
-    if ( ! IS_MUTABLE_OBJ(list) ) {
-        return list;
-    }
```

Comment:
Huh... Hadn't thought about this before, but shouldn't this check for (im)mutable stay in? It makes sense to me from a logical point of view, and I just checked, this is also what we do for plists. @stevelinton any reason why your original PR removed this?!?